a3fe/tests/test_engine_configuration.py

```diff
@@ -183,7 +207,7 @@ def test_copy_from_existing_config():
 
     assert c.use_boresch_restraints is True
     assert c.turn_on_receptor_ligand_restraints is False
-    assert c.topfile.endswith("somd.prm7")
+    assert c.boresch_restraints_dictionary is not None
```

Comment:
Please add a test where you:

- Read in the config file with _from_config_file
- Write out the config again
- Check that every line in the original config file is in the new config file

This will confirm that no information is lost. Thanks